### PR TITLE
[#2936] Add Kafka broker configuration guide

### LINKED ISSUE #2936: Add Kafka broker configuration guide
It would be helpful to have a separate guide on which kinds of Kafka topics are used by Hono and how to configure them.
This involves for example the retention configuration.
Also notes on the general broker configuration may be helpful.

This relates to #2931 (adding a hint to choose a rather short retention period for the command topic).

### PR DESCRIPTION
**Pull request by calohmn**
Also apply corrections to the messaging sections of the other admin guides.

This fixes #2936.

### CODE REVIEW

File: site/documentation/content/admin-guide/mongodb-device-registry-config.md [hunk 1]
```diff
@@ -128,3 +128,29 @@ The file needs to contain at least all versions of the key that values in the da
 Otherwise the registry will not be able to decrypt these values during reading. The `defaultKey` property indicates the
 version of the key that should be used when encrypting values. Please refer to the
 [CryptVault project](https://github.com/bolcom/cryptvault) for additional information regarding key rotation.
+
+## Messaging Configuration
```
Comment (sophokles73) [inline site/documentation/content/admin-guide/mongodb-device-registry-config.md:132]: what about the notification configuration?

File: site/documentation/content/admin-guide/jdbc-device-registry-config.md [hunk 2]
```diff
@@ -129,3 +129,29 @@ for more information.
 
 See [Monitoring & Tracing Admin Guide]({{< relref "monitoring-tracing-config.md" >}}) for details on how to configure
 the reporting of metrics.
+
+## Messaging Configuration
```
Comment (sophokles73) [inline site/documentation/content/admin-guide/jdbc-device-registry-config.md:133]: what about the notification configuration?

File: site/documentation/content/admin-guide/file-based-device-registry-config.md [hunk 3]
```diff
@@ -246,3 +246,29 @@ device that connects directly to the adapter.
 
 The example configuration file (located at `$HONO/services/device-registry-file/src/test/resources/device-identities.json`)
 includes a device and a corresponding gateway configured in this way.
+
+## Messaging Configuration
```
Comment (sophokles73) [inline site/documentation/content/admin-guide/file-based-device-registry-config.md:250]: what about the notification configuration?

File: site/documentation/content/admin-guide/command-router-config.md [hunk 4]
```diff
@@ -107,19 +107,35 @@ This can be used to narrow the visibility of the insecure port to a local networ
 
 Both the secure as well as the insecure port numbers may be explicitly set to `0`. The Command Router component will then use arbitrary (unused) port numbers determined by the operating system during startup.
 
-## Command & Control Connection Configuration
+## AMQP 1.0 Messaging Network Connection Configuration
```
Comment (sophokles73) [inline site/documentation/content/admin-guide/command-router-config.md:110]: what about the notification configuration?

File: site/documentation/content/admin-guide/mongodb-device-registry-config.md (hunk 1 above)
Comment (calohmn) [reply site/documentation/content/admin-guide/mongodb-device-registry-config.md:132]: Currently the messaging configuration is used for that (in the AMQP case) - see https://github.com/eclipse/hono/pull/3019#discussion_r782191263.

Comment (calohmn) [reply site/documentation/content/admin-guide/mongodb-device-registry-config.md:132]: I've modified the whole `Messaging Configuration` chapter on all relevant admin guides here accordingly.

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**